coderd/templateversions.go

```diff
@@ -523,15 +522,15 @@ func (api *API) templateVersionsByTemplate(rw http.ResponseWriter, r *http.Reque
 				})
 				return err
 			}
-			createdByName, err := getUsernameByUserID(ctx, store, version.CreatedBy)
+			user, err := api.Database.GetUserByID(ctx, version.CreatedBy)
```

Comment:
@BrunoQuaresma this is the line you need to change. Make this `store.GetUserByID`

Reply:
I would also remove this transaction or consult with whoever wrote it to ensure that it is necesssary

Reply:
Fixed! 👍